commands/jail.js

```diff
@@ -2,39 +2,41 @@ const { SlashCommandBuilder, ButtonBuilder, ButtonStyle, ActionRowBuilder, Embed
 const rulesKimoState = require('../patterns/rulesKimoState');
 const Jail = require('../models/jailTracker');
 const jail = require('../patterns/jail');
+const { jailedRoleID, adminRoleID } = require('../ids.json');
 
 module.exports = {
     data: new SlashCommandBuilder()
-    .setName('jail')
-    .setDescription('jail someone.')
-    .addUserOption(option =>
-        option
-            .setName('target')
-            .setDescription('to jail')
-            .setRequired(true))
-    .addIntegerOption(option =>
-        option
-            .setName('time')
-            .setDescription('minutes to jail')
-            .setRequired(true))
-    .addStringOption(option =>
-        option
-            .setName('reason')
-            .setDescription('reason for jailing')
-            .setRequired(true)),
+        .setName('jail')
```

Comment:
Since the all of these lines that start with `.` are being done on the `SlashCommandBuilder()` object, indenting them makes it more clear that they're part of the line above